swift_browser_ui/ui/server.py

```diff
@@ -82,12 +85,24 @@ async def servinit(
     app = aiohttp.web.Application(middlewares=middlewares)  # type: ignore
 
     # Initialize aiohttp_session
+    redis_creds = ""
+    redis_user = str(os.environ.get("SWIFT_UI_REDIS_USER", ""))
```

Comment:
if we require these, there might be an option to omit the default so that server does not start if the variables are not set.

Reply:
None of these are strictly required, a redis container with defaults will work when you just start the browser backend. These are to allow running on e.g. Rahti where credentials and host information are a must.